modules/virt-expanding-pvc-with-data-volumes.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-expanding-virtual-storage-with-blank-disk-images.adoc
+// * virt/virtual_machines/virtual_disks/virt-expanding-vm-disks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-creating-blank-disk-datavolumes_{context}"]
-= Creating a blank disk image with data volumes
+[id="virt-expanding-pvc-with-data-volumes_{context}"]
+= Expanding a PVC by adding data volumes
 
-You can expand a persistent volume claim by creating a data volume for a blank disk image.
+You can expand a persistent volume claim (PVC) by adding a blank data volume.
```

Comment:
So this sentence makes no sense. You add disk space to a Virtual machine by either increasing the size of an existing disk (expand PVC size) OR you add an additional disk by adding a blank data volume, something like this

`You can expand available storage of a Virtual Machine, by adding a blank data volume`